service/service.go

```diff
@@ -54,6 +54,17 @@ type Host interface {
 	// GetServiceConsumers gets the list of services which depend on a service by
 	// name.
 	GetServiceConsumers(serviceName string) []Consumer
+
+	// NewController returns an unstarted, isolated Controller that a Service
+	// can use to instantiate its own components.
+	NewController(id string) Controller
```

Comment:
I wonder about this one and if we need to do some additional work to make running multiple flow controllers a smooth experience if services can also start them. I know there's already some limitations in the debug UI for graphing modules 1.0.

Reply:
Ideally yes. I'd like at least to explore with what can be displayed in the UI (even if it's eg. a new page just for remotecfg).